sdk/network/Azure.ResourceManager.Network/src/Generated/VirtualHubRouteTableV2Collection.cs

```diff
@@ -51,7 +51,7 @@ internal VirtualHubRouteTableV2Collection(ArmResource parent) : base(parent)
         /// <param name="waitForCompletion"> Waits for the completion of the long running operations. </param>
         /// <param name="cancellationToken"> The cancellation token to use. </param>
         /// <exception cref="ArgumentNullException"> <paramref name="routeTableName"/> or <paramref name="virtualHubRouteTableV2Parameters"/> is null. </exception>
-        public virtual VirtualHubRouteTableV2CreateOrUpdateOperation CreateOrUpdate(string routeTableName, VirtualHubRouteTableV2Data virtualHubRouteTableV2Parameters, bool waitForCompletion = true, CancellationToken cancellationToken = default)
+        public virtual VirtualHubRouteTableV2SCreateOrUpdateOperation CreateOrUpdate(string routeTableName, VirtualHubRouteTableV2Data virtualHubRouteTableV2Parameters, bool waitForCompletion = true, CancellationToken cancellationToken = default)
```

Comment:
This change is a known issue, we will fix this in future changes

Reply:
Is there an issue tracking it?

Reply:
Yes, actually this will be fixed by https://github.com/Azure/autorest.csharp/issues/1719